src/style.css

```diff
@@ -0,0 +1,42 @@
+body {
+  margin: auto;
+}
+
+.stars {
+  display: flex;
+  margin-bottom: 0;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 2px;
+  margin-left: 2px;
+  background-position: center;
+  background-repeat: no-repeat;
+  flex-shrink: 0;
+}
+
+.stars__star:last-child {
+  margin-right: 2px;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars--0 .stars__star:nth-child(-n+5),
+.stars--1 .stars__star:nth-child(n+2),
+.stars--2 .stars__star:nth-child(n+3),
+.stars--3 .stars__star:nth-child(n+4),
+.stars--4 .stars__star:nth-child(n+5) {
+  background-image: url(./images/star.svg);
+}
+
```

Comment:
it looks redundant, you can just set the background-image in .stars__star and overwrite it here
<img width="509" alt="image" src="https://github.com/mate-academy/layout_stars/assets/70220667/1f9b971d-4e62-411d-9b8a-ae4b9f6da066">